SPECS/autogen/autogen.spec

```diff
@@ -1,21 +1,22 @@
-Summary:	The Automated Text and Program Generation Tool
-Name:		autogen
-Version:    5.18.16
-Release:    7%{?dist}
+Summary:        The Automated Text and Program Generation Tool
+Name:           autogen
+Version:        5.18.16
+Release:        8%{?dist}
 License:        GPLv3+
 URL:            http://www.gnu.org/software/autogen/
 Source0:        ftp://ftp.gnu.org/gnu/autogen/rel%{version}/%{name}-%{version}.tar.xz
```

Comment:
Can you switch the source to the HTTPS mirror? We've had requests from our component governance people to eliminate the FTP source URLs.